ToNote/ViewModels/MainViewModel.cs

```diff
@@ -99,6 +99,22 @@ public ICommand RemoveNoteCommand
                     var metadataFileName = note.Name + "Metadata.txt";
                     if (File.Exists(metadataFileName))
                         File.Delete(metadataFileName);
+
+                    var todosFileName = "*_" + note.Name + "_TODO";
+
+                    try
+                    {
+                        string[] todoFiles = Directory.GetFiles("Data", todosFileName);
```

Comment:
Use var, or even better, inline the `Directory.GetFiles("Data", todosFileName)` call into the `foreach`